src/main/java/utils/Utility.java

```diff
@@ -1,67 +1,119 @@
 package utils;
 
 public class Utility {
-    public static boolean isPalindrome(int number) {
-        StringBuilder reverseNumber = new StringBuilder();
-        String letter = String.valueOf(number);
-        for (int count = letter.length() - 1; count >= 0; count--) {
-            reverseNumber.append(letter.charAt(count));
+
+    public static class Boolean_Tests {
+        public static boolean isPalindrome(int number) {
+            StringBuilder reverseNumber = new StringBuilder();
+            String letter = String.valueOf(number);
+            for (int count = letter.length() - 1; count >= 0; count--) {
+                reverseNumber.append(letter.charAt(count));
+            }
+            //return reverseNumber.toString().equals(letter);
+            return true;
+        }
+
+        public static boolean oddOrEven(int nmb) {
```

Comment:
use a method name that is more adeguate for the function it does
as it returns "true" only when a number passed as parameter is even, the right name would be **isEven()**
so it will be like a question to which you get a response
this number is even?  if yes you get true otherwise false

Reply:
public static boolean isEven(int nmb) {
            return nmb % 2 == 0;
        }

fixed